src/func.jl

```diff
@@ -169,17 +176,17 @@ function FunctionOperator(op,
                          traits,
                          p,
                          t,
-                         cache,
+                         normalize_kwargs(kwargs_for_op),
+                         cache
                         )
 
     ifcache ? cache_operator(L, input, output) : L
 end
 
-function update_coefficients(L::FunctionOperator, u, p, t)
-    op = update_coefficients(L.op, u, p, t)
-    op_adjoint = update_coefficients(L.op_adjoint, u, p, t)
-    op_inverse = update_coefficients(L.op_inverse, u, p, t)
-    op_adjoint_inverse = update_coefficients(L.op_adjoint_inverse, u, p, t)
+function update_coefficients(L::FunctionOperator, u, p, t; kwargs...)
+    for op in getops(L)
+        op = update_coefficients(op, u, p, t; kwargs...)
+    end
```

Comment:
since `update_coeffecients()` is oop, you gotta store the output.
```julia
new_ops = ()
for op in getops(L)
    op = update_coefficients(op, u, p, t; kwargs...)
    new_ops = (new_ops..., op)
end

L = FunctionOperator(new_ops..., L.traits, etc)
```

Reply:
good catch, reverted to what you had originally (but with kwargs)